### ISSUE TITLE
Headers overwritten by cors_headers

### ISSUE THREAD

**Issue by Helveg** [essenciary/Genie.jl#120]
Whenever the `cors_headers` dict is applied it seems all my other headers are removed:
```
route("*", method = "OPTIONS") do
  HTTP.Response(200, ["X-Test-Header" => "Anything", "Access-Control-Allow-Origin" => "*"], body = "Preflight OK")
end
```

![image](https://user-images.githubusercontent.com/28923979/53164068-b7395100-35cf-11e9-870f-735ceae8bd80.png)

The "Preflight OK" body **IS** sent as response.

**Comment by essenciary** [comment]
@Helveg Thanks for reporting this - it should now be fixed in 0.8.5

**Comment by essenciary** [comment]
If that is not the case, please reopen the issue with the extra details.
